python/pdstools/utils/cdh_utils.py

```diff
@@ -266,20 +280,30 @@ def get_latest_file(path: str, target: str, verbose: bool = False) -> str:
         return f"Target not found"
 
     supported = [".json", ".csv", ".zip", ".parquet", ".feather", ".ipc", ".arrow"]
+    try:
+        files_dir = [
+            f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))
+        ]
+        files_dir = [
+            f for f in files_dir if os.path.splitext(f)[-1].lower() in supported
+        ]
+    except FileNotFoundError as e:
```

Comment:
Is this right? FileNotFound doesn't sound to me like it's a directory not found?

Reply:
Right. So what I mean is you raise a FileNotFoundError, but in the text you mention that the directory does not exist. That's typically not very verbose - and if I got a FileNotFoundError and the text mentioned that the directory does not exist - I wouldn't know what to believe